environments/icds-cas/public.yml

```diff
@@ -31,6 +31,7 @@ nofile_limit: 65536
 redis_maxmemory: 6gb
 redis_maxmemory_policy: allkeys-lfu
 redis_auto_aof_rewrite_percentage: "50"
+redis_password: "{{ localsettings_private.REDIS_PASSWORD }}"
```

Comment:
i'm not quite sure about this but i think this will work https://github.com/DavidWittman/ansible-redis/blob/master/defaults/main.yml#L34